src/Sarif.Sarifer/TextBufferViewTracker.cs

```diff
@@ -18,26 +19,31 @@ namespace Microsoft.CodeAnalysis.Sarif.Sarifer
     [Export(typeof(ITextBufferViewTracker))]
     public class TextBufferViewTracker : ITextBufferViewTracker
     {
-        private readonly IDictionary<ITextBuffer, TextBufferViewTrackingInformation> bufferToViewsDictionary = new Dictionary<ITextBuffer, TextBufferViewTrackingInformation>();
+        protected const int DefaultUpdateDelayInMS = 1500;
```

Comment:
>protected const int DefaultUpdateDelayInMS = 1500; [](start = 8, length = 50)

remove this since we are not using

Reply:
done

---
In reply to: [564965399](https://github.com/microsoft/sarif-visualstudio-extension/pull/349#discussion_r564965399) [](ancestors = 564965399)